### ISSUE TITLE
CasperJS support?

### ISSUE THREAD

**Issue by moshewe** [medialab/sandcrawler#188]
Would you consider supporting CasperJS in the future? It's basically a PhantomJS instance, but with some scraping and navigation built-in.

**Comment by Yomguithereal** [comment]
Would the main point to have access to easy navigation?

**Comment by moshewe** [comment]
Yes, but also scraping infrastructure. It also implements XPath selectors and some CSS selectors so that you don't have to go through `page.evaluate` and such. Interacting with the page is also made easier.

It's a library built on top of phantom, basically.
